openmetadata-ui/src/main/resources/ui/src/components/Modals/WhatsNewModal/index.tsx

```diff
@@ -69,126 +67,109 @@ export const WhatsNewModal: FunctionComponent<Props> = ({
   }, [activeData]);
 
   return (
-    <dialog className="tw-modal" data-testid="whats-new-dialog">
-      <div className="tw-modal-backdrop" />
-      <div
-        className="tw-modal-container tw-pb-0"
-        style={{ maxWidth: '1050px' }}>
-        <div className="tw-modal-header">
-          <p className="tw-modal-title">{header}</p>
-
-          <div className="tw-flex">
-            <svg
-              className="tw-w-6 tw-h-6 tw-ml-1 tw-cursor-pointer"
-              data-testid="closeWhatsNew"
-              fill="none"
-              stroke="currentColor"
-              viewBox="0 0 24 24"
-              xmlns="http://www.w3.org/2000/svg"
-              onClick={handleCancel}>
-              <path
-                d="M6 18L18 6M6 6l12 12"
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                strokeWidth="2"
-              />
-            </svg>
+    <Modal
+      centered
+      destroyOnClose
+      className="what-news-modal"
+      closeIcon={
+        <CloseWhatNewIconComponent
+          dataTestId="closeWhatsNew"
+          handleCancel={handleCancel}
+        />
+      }
+      data-testid="whats-new-dialog"
+      footer={null}
+      title={
+        <Typography.Text strong data-testid="what-news-header">
+          {header}
+        </Typography.Text>
+      }
+      visible={visible}
+      width={1200}>
+      <div className="flex w-auto h-full">
+        <div
+          className="border-r-2 p-x-md p-y-md border-separate"
+          style={{ width: '14%' }}>
+          <div className="flex flex-col-reverse">
```

Comment:
Can we use antd 'Space' or 'Row/Col' here?

Reply:
@aniketkatkar97 , We can do that but in a separate PR as this PR is for replacing native dialog with antd modal.